string gamestate
string[] playernames
int32[] playerstates
int32[] playerscores
